priv/templates/phx.gen.live/live_helpers.ex

```diff
@@ -19,7 +19,7 @@ defmodule <%= inspect context.web_module %>.LiveHelpers do
           title={@page_title}
           action={@live_action}
           return_to={Routes.<%= schema.singular %>_index_path(@socket, :index)}
-          <%= schema.singular %>: @<%= schema.singular %>
+          <%= schema.singular %>={@<%= schema.singular %>}
```

Comment:
I am trying to change this in another PR, too.

The generated code is something like:

```
    <.live_component
      module={AdminWeb.VersionLive.FormComponent}
      id={@version.id}
      title={@page_title}
      action={@live_action}
      version={@version} <- schema.singular
      return_to={Routes.version_show_path(@socket, :show, @version)}
    />
```

As you can see, `schema.singular` comes before `return_to`. I think it would be better to adjust the order. Then, I can close my own PR. ;)

Reply:
> I think it would be better to adjust the order

Not sure if I follow it. Would you mind providing the reasoning behind the ordering change in #4704?

Reply:
[The template for generating the final code](https://github.com/phoenixframework/phoenix/blob/bc6ece3a632552f2114441d4ff306f7d044ff0fe/priv/templates/phx.gen.live/show.html.heex#L10) is using this order.

Reply:
Got it. Sure it maybe could use some adjustments. :+1: